real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/main/RobobuggyConfigFileTest.java

```diff
@@ -0,0 +1,206 @@
+package com.roboclub.robobuggy.main;
+
+import com.roboclub.robobuggy.robots.PlayBackRobot;
```

Comment:
What you can do in the future is right-click on `src` and click `Optimize Imports...`. Strange that checkstyle doesn't catch that